xml/rmt_rmt-cli.xml

```diff
@@ -258,6 +272,60 @@
      </para>
     </listitem>
    </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli mirror all</command></term>
+    <listitem>
+     <para>
+      This command mirrors all enabled repositories.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli mirror repositories [IDs]</command></term>
+    <listitem>
+     <para>
+      This command mirrors enabled repositories by a list of IDs.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli mirror products [IDs]</command></term>
+    <listitem>
+     <para>
+      This command mirrors enabled repositories for a product by a list of IDs.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+ </sect2>
+
+ <sect2 xml:id="sec-rmt-tools-rmt-cli-systems">
+  <title><command>systems</command></title>
+  <variablelist>
+   <varlistentry>
+    <term><command>rmt-cli systems list</command></term>
+    <listitem>
+     <para>
+      This command lists registered systems.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli systems scc-sync</command></term>
+    <listitem>
+     <para>
+      This command forward registered systems data to SCC.
+     </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry>
+    <term><command>rmt-cli systems remove [TARGET]</command></term>
+    <listitem>
+     <para>
+      This command removes a system from RMT by its login.
```

Comment:
i dont understand what "by its login" means. Can you maybe share more words that would explain it?

Reply:
HI @tbazant, `login` is a column in the systems list:

```
bin/rmt-cli systems list
```

```                                                                                                                                                                                                    Tue 3 10:03AM
+--------------------------------------+------------+-------------------------+-------------------------+------------------------------------------+
| Login                                | Hostname   | Registration time       | Last seen               | Products                                 |
+--------------------------------------+------------+-------------------------+-------------------------+------------------------------------------+
| SCC_bc0d2a200d884bddaf0f51e31165193b | linux-h8ph | 2020-09-08 15:42:04 UTC | 2020-09-08 15:42:29 UTC | SLES/15/x86_64                           |
|                                      |            |                         |                         | sle-module-basesystem/15/x86_64          |
|                                      |            |                         |                         | sle-module-server-applications/15/x86_64 |
| SCC_476bbc7fa9d54c0b8ae655c0f98ebbc5 | linux-h8ph | 2020-09-08 15:40:55 UTC | 2020-09-08 15:41:22 UTC | SLES/15/x86_64                           |
|                                      |            |                         |                         | sle-module-basesystem/15/x86_64          |
|                                      |            |                         |                         | sle-module-server-applications/15/x86_64 |
| SCC_ee59a04ad6fe4b05a056b52a402452ea | linux-h8ph | 2020-08-28 09:05:57 UTC | 2020-08-28 09:07:02 UTC |                                          |
| SCC_8415767e76ca47978372cc2cc8011f7a | linux-h8ph | 2020-08-28 08:36:21 UTC | 2020-08-28 09:04:08 UTC |                                          |
| SCC_7f14501fc6a14a5ca17c5eaa247d9178 | linux-h8ph | 2020-08-27 15:15:10 UTC | 2020-08-27 15:33:25 UTC |                                          |
| SCC_553b936873664ba1a302590ca17b30b4 | linux-h8ph | 2020-08-27 15:09:32 UTC | 2020-08-27 15:14:32 UTC |                                          |
| SCC_a7e0172feb6a4b949052910ee02b50c2 | linux-h8ph | 2020-08-27 15:08:04 UTC | 2020-08-27 15:09:07 UTC |                                          |
| SCC_b103739340884eee8c415d9f7a62c41e | linux-h8ph | 2020-08-27 15:06:50 UTC | 2020-08-27 15:06:52 UTC |                                          |
| SCC_34dfcff7108448c4afb2ca8d648cda80 | linux-h8ph | 2020-08-27 15:06:32 UTC |                         |                                          |
| SCC_8c0cb836facc4aad9f90591ac2f5322e | linux-h8ph | 2020-08-27 13:51:49 UTC | 2020-08-27 13:52:02 UTC |                                          |
| SCC_6f133f6e8c444c2b8a6175a99ba9ca3f | linux-h8ph | 2020-08-27 13:41:30 UTC | 2020-08-27 13:47:12 UTC |                                          |
| SCC_4163f3ae9a164dbfaea3793e689c6a76 | linux-h8ph | 2020-08-27 13:40:44 UTC |                         |                                          |
| SCC_f006c8ca9abd484ab294008a5b1e7ef6 | linux-h8ph | 2020-08-27 13:36:40 UTC |                         |                                          |
| SCC_a64ff681b97f489ba8b13d1bf3443792 | linux-h8ph | 2020-08-27 13:34:12 UTC | 2020-08-27 13:34:18 UTC |                                          |
| SCC_746e3cd7e30f4d4ca67db686b5765af4 | linux-h8ph | 2020-08-27 13:33:41 UTC | 2020-08-27 13:33:42 UTC |                                          |
| SCC_f11f91b0bdf344c8a9dede45cc558b00 | linux-h8ph | 2020-08-27 13:32:42 UTC | 2020-08-27 13:32:43 UTC |                                          |
| SCC_ee46a677523547688423cc4fcf97c9bb | linux-h8ph | 2020-08-27 13:31:48 UTC | 2020-08-27 13:31:50 UTC |                                          |
| SCC_c2509174f42b43d29298930f38f7f057 | linux-h8ph | 2020-08-27 13:30:15 UTC | 2020-08-27 13:30:17 UTC |                                          |
| SCC_8257dc86dfd34cbfa0e2142ab8b84991 | linux-h8ph | 2020-08-27 13:29:43 UTC | 2020-08-27 13:29:44 UTC |                                          |
| SCC_5ad740e7146e431cb582e8e1115971c0 | linux-h8ph | 2020-08-27 13:15:39 UTC | 2020-08-27 13:15:40 UTC |                                          |
+--------------------------------------+------------+-------------------------+-------------------------+------------------------------------------+
Showing last 20 registrations. Use the '--all' option to see all registered systems.
```

When a system registers to RMT, RMT will create for it a login/password combination and pass it along. The login is the unique identifier that RMT shows to users as it is also the most useful to have.